server/src/main/java/org/elasticsearch/cluster/ClusterInfo.java

```diff
@@ -191,10 +203,31 @@ public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params params
                 }
                 return builder.endObject(); // NodeAndPath
             }),
-            endArray() // end "reserved_sizes"
+            chunk(
+                (builder, p) -> builder.endArray() // end "reserved_sizes"
+                    .startObject("heap_usage")
+            ),
+            Iterators.map(nodesHeapUsage.entrySet().iterator(), c -> (builder, p) -> {
+                builder.startObject(c.getKey());
+                c.getValue().toShortXContent(builder);
+                builder.endObject();
+                return builder;
+            }),
+            endObject() // end "heap_usage"
```

Comment:
We probably don't want include the new heap usage in the xcontent just yet? This will be a visible stateful change via the AllocationExplain API. Since we may still want to iterate on this field once we get more data from serverless, not including it here gives us flexibility.

Reply:
Removed in 85fd019c84e47f2ca09c5c071eeeb5d297493c4f